articles/101_build_tool.md

```diff
@@ -289,59 +290,168 @@ The following items are possible extension points to provide custom functionalit
+Since `catkin_tools` is in many aspects the most complete ROS build tool it should be the one being evolved.
 While `ament_tools` has a few features `catkin_tools` currently lacks (e.g. plain CMake support without a manifest, Windows support) the feature richness of `catkin_tools` makes it a better starting point.
 
-The following items are highlighting some of the necessary efforts (not a complete list):
+### Start "from scratch" / colcon
+
+Since the first draft of this article the `colcon` project has been developed with the goals and requirements of a universal build tool in mind.
+In its current form it is already able to build ROS 1 workspaces, ROS 2 workspaces, as well as Gazebo including its ignition dependencies.
+It uses Python 3.5+ and targets all platforms supported by ROS: Linux, macOS, and Windows.
+
+Since it hasn't been used by many people yet more advanced features like cross compilation, `DESTDIR`, etc. hasn't been tested (and will therefore likely not work yet).
+
+## Decision process
+
+For the decision process only the following two options are being considering based on the rationale described above:
+
+A.  Use `catkin_tools` as a starting point
+B.  Use `colcon` as a starting point
+
+If this topic would have been addressed earlier some of the duplicate effort could have likely been avoided.
+When the work towards a universal build tool was suspended over a year ago it was a conscious decision based on available resources.
+Nevertheless move forward with a decision now will at least avoid further uncertainty and duplication.
+
+Both of the considered options have unique and valuable features and there are good arguments to build our future development on either of the two tools.
+Since both are written in Python either of the two tools could be "transformed" to cover the pros of the other one.
+So the two important criteria for the decision are:
+
+- the effort it takes to do (in the short term as well as in the long term) and
+- the difference of the resulting code base after the "transformation" is completed.
+
+### Immediate goals
+
+A ROS 2 developer currently builds a steadily growing workspace with ROS 2 packages.
+The same is happening in the monolithic Jenkins jobs on [ci.ros2.org](https://ci.ros2.org) (with the advantage to test changes across repositories easily).
+Therefore features to easily filter the packages which need to be build are eagerly awaited to improve the development process.
+
+For the last ROS 2 release *Ardent* the buildfarm [build.ros.org](http://build.ros2.org) only provides jobs to generate Debian packages.
+Neither *devel* jobs or *pull request* jobs are available nor is it supported to build a local *prerelease*.
+For the coming ROS 2 release *Bouncy* these job types should be available to support maintainers.
+
+In ROS 2 *Bouncy* the universal build tool will be the only supported option and `ament_tools` will be archived.
```

Comment:
In Bouncy, while encouraging users to use the universal build tool, `ament_tools` will still be released in Bouncy to ease user transition and the repository cannot be archived until the distributions using it are all EOL (in this case June 2019)

Reply:
Updated in 81b8dbd43ef64c29dcdfc2a6e11246ff4775eb44.